website/content/v1.9/talos-guides/configuration/image-cache.md

```diff
@@ -79,21 +82,22 @@ machine:
 
 Once enabled, Talos Linux will automatically look for the image cache contents either on the disk or in the ISO image.
 
-If the image cache is bundled with the ISO image, the disk volume for the image cache should be configured in the Talos configuration to copy the image cache to the disk during the installation process:
+If the image cache is bundled with the ISO, the disk volume size for the image cache should be configured to copy the image cache to the disk during the installation process:
 
 ```yaml
 apiVersion: v1alpha1
 kind: VolumeConfig
 name: IMAGECACHE
 provisioning:
   diskSelector:
-    match: 'system_disk'
+    match: "system_disk"
   minSize: 2GB
   maxSize: 2GB
 ```
 
-In this example, image cache volume is provisioned on the system disk with a fixed size of 2GB.
+In this example, image cache volume is provisioned on the system disk with a fixed size of 2GB (default is 1GB).
```

Comment:
default is actually 1.3Gb

Reply:
the actual defaults are: https://github.com/siderolabs/talos/blob/470b75563add4ce5bbce312c1e3dc783e63af1fa/internal/app/machined/pkg/controllers/cri/image_cache_config.go#L102-L104

Reply:
How does it pick between min and max?

Reply:
it picks a value in between leaning towards max if space is available, but it will never allocated less than min